.github/workflows/main.yml

```
@@ -27,6 +29,51 @@
+          pkill Xquartz || true
+          pkill Xvfb || true
+          sleep 2
+          
+          # Configure XQuartz
+          defaults write org.x.X11 enable_test_extensions -boolean true
+          defaults write org.x.X11 app_to_run /opt/X11/bin/xterm
+          defaults write org.x.X11 no_auth -boolean true
+          defaults write org.x.X11 nolisten_tcp -boolean false
+
+      - name: Start X server
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2009:info:10:1: Consider using pgrep instead of grepping ps output

(shellcheck)